### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by ninasalazar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # google-homepage
 Google Homepage
 http://www.theodinproject.com/web-development-101/html-css?ref=lnav
+Nina test
```
Comment (redhedjim) [inline README.md]: @ninasalazar I need more information. Please tell me what you're testing.

Comment (ninasalazar) [reply README.md]: updated file

Comment (redhedjim) [reply README.md]: Thanks!

**Review by redhedjim — APPROVED** [review]
This looks great! Perfect testing.

**Pull request merged**